### PR TITLE
PLANET-3879 Add articles block

### PR DESCRIPTION
**Pull request by kirdia**
First version of articles block

### CODE REVIEW

File: react-blocks/src/blocks/Articles/Articles.js [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    const {__} = wp.i18n;
+
+    const tagSuggestions = this.props.tagsList.map(tag => tag.name);
+    const postTypeSuggestions = this.props.postTypesList.map(postType => postType.name);
+
+    return (
+      <Fragment>
+        <div>
+          <TextControl
+            label="Title"
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/Articles.js]: Need to use translation function here.

File: react-blocks/src/blocks/Articles/Articles.js [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+            help={__('Your default is set to', 'planet4-blocks-backend')}
+            value={this.props.title}
+            onChange={this.props.onTitleChange}
+          />
+        </div>
+
+
+        <div>
+          <TextareaControl
+            label="Description"
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/Articles.js]: Same here and rest of labels and placeholders as well

File: react-blocks/src/blocks/Articles/Articles.js [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+                  label={__('Select Tags', 'p4ge')}
+                  onChange={tokens => this.onSelectedTagsChange(tokens)}
+                />
+                <p className='FieldHelp'>Associate this block with Actions that have specific Tags</p>
+              </div>
+              <div>
+                <FormTokenField
+                  value={this.state.postTypeTokens}
+                  suggestions={postTypeSuggestions}
+                  label='Post Types'
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/Articles.js]: Need to use i18n function.

File: react-blocks/src/blocks/Articles/Articles.js [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+              </div>
+            </Fragment>
+            : null
+        }
+
+        {
+          (this.props.tags.length === 0 && this.props.post_types.length === 0)
+            ? <div>
+              <hr/>
+              <label>Manual override</label>
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/Articles.js]: Need to use string translation function here.

File: react-blocks/src/blocks/Articles/ArticlesBlock.js [hunk 5]
```diff
@@ -0,0 +1,232 @@
+// import { CoversIcon } from './CoversIcon.js';
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/ArticlesBlock.js]: Please remove commented code

**Comment by comzeradd** [comment]
@sagarsdeshmukh PR updated

File: react-blocks/src/blocks/Articles/ArticlesBlock.js [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+          type: 'array',
+          default: []
+        },
+        post_types: {
+          type: 'array',
+          default: []
+        },
+        read_more_text: {
+          type: 'string',
+          default: '1'
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/ArticlesBlock.js]: I guess the default value is not needed at here.

File: react-blocks/src/blocks/Articles/Articles.js [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+            value={this.props.description}
+            onChange={this.props.onDescriptionChange}
+          />
+        </div>
+
+        <div>
+          <TextControl
+            label={__('Button Text', 'p4ge')}
+            placeholder={__('Override button text', 'p4ge')}
+            help={__('Your default is set to', 'p4ge')}
```
Comment (sagarsdeshmukh) [inline react-blocks/src/blocks/Articles/Articles.js]: could you update the help text with `Your default is set to [ Load More ]`

**Comment by sagarsdeshmukh** [comment]
@comzeradd - I added comment for small changes, except that looks good to me 👍

**Comment by comzeradd** [comment]
@sagarsdeshmukh good catch! PR amended.

**Pull request merged**